x-pack/plugin/security/qa/tls-basic/build.gradle

```diff
@@ -35,15 +34,10 @@ integTestCluster {
   setting 'xpack.security.http.ssl.certificate', 'http.crt'
   setting 'xpack.security.http.ssl.key', 'http.key'
   setting 'xpack.security.http.ssl.key_passphrase', 'http-password'
+  setting 'xpack.security.http.ssl.certificate_authorities', 'ca.crt'
```

Comment:
I'm not sure why this wasn't here before , but it made more sense to teach test-clusters about this one rather than transport.